### ISSUE TITLE
Parser selection --parsers seems to be ignored

### ISSUE THREAD

**Issue by kinky-it** [log2timeline/plaso#2732]
**Description of problem:**

When running `log2timeline.py` with the `--parsers` parameter, the parameter value seems to be ignored. The automatic parser selection seems to override the specified parser selection. 

Information from log file and `pinfo.py` output is mentioned below. The same happens with parser exclusion (`--parsers !winreg`) as well as on parser selection (`--parsers winreg`).

**Command line and arguments:**

```
--parsers !winreg
--parsers winreg
```

**Source data:**

Input E01 file that needs parsing was successfully limited to a single SYSTEM registry file. 

**Plaso version:**

plaso - log2timeline version 20190708

**Operating system Plaso is running on:**

Fedora release 30 (Thirty)

**Installation method:**

RPM plaso-20190708-1.fc30.x86_64 from LiFTeR ( https://forensics.cert.org/ )

**Debug output/tracebacks:**

(Selection of) Output from pinfo.py:

```
Parser filter expression :
                             amcache,bencode,binary_cookies,chrome_cache,chrome_preferences,custom_destinations,czip/oxml,esedb,esedb/file_history,esedb/msie_webcache,filestat,firefox_cache,gdrive_synclog,java_idx,lnk,mcafee_protection,msiecf,olecf,olecf/olecf_automatic_destinations,opera_global,opera_typed_history,pe,plist/safari_history,prefetch,recycle_bin,sccm,skydrive_log,skydrive_log_old,sqlite/chrome_27_history,sqlite/chrome_8_history,sqlite/chrome_autofill,sqlite/chrome_cookies,sqlite/chrome_extension_activity,sqlite/firefox_cookies,sqlite/firefox_downloads,sqlite/firefox_history,sqlite/google_drive,sqlite/skype,symantec_scanlog,usnjrnl,winevtx,winfirewall,winjob,winreg
Enabled parser and plugins : amcache, bencode, bencode/bencode_transmission,
                             bencode/bencode_utorrent, binary_cookies,
                             chrome_cache, chrome_preferences,
                             custom_destinations, czip, czip/oxml, esedb,
                             esedb/file_history, esedb/msie_webcache,
                             esedb/srum, filestat, firefox_cache,
                             gdrive_synclog, java_idx, lnk, mcafee_protection,
                             msiecf, olecf, olecf/olecf_automatic_destinations,
                             olecf/olecf_default, olecf/olecf_document_summary,
                             olecf/olecf_summary, opera_global,
                             opera_typed_history, pe, plist, plist/airport,
                             plist/apple_id, plist/ipod_device,
                             plist/macosx_bluetooth,
                             plist/macosx_install_history, plist/macuser,
                             plist/maxos_software_update, plist/plist_default,
                             plist/safari_history, plist/spotlight,
                             plist/spotlight_volume, plist/time_machine,
                             prefetch, recycle_bin, sccm, skydrive_log,
                             skydrive_log_old, sqlite, sqlite/android_calls,
                             sqlite/android_sms, sqlite/android_webview,
                             sqlite/android_webviewcache, sqlite/appusage,
                             sqlite/chrome_27_history, sqlite/chrome_8_history,
                             sqlite/chrome_autofill, sqlite/chrome_cookies,
                             sqlite/chrome_extension_activity,
                             sqlite/firefox_cookies, sqlite/firefox_downloads,
                             sqlite/firefox_history, sqlite/google_drive,
                             sqlite/hangouts_messages, sqlite/imessage,
                             sqlite/kik_messenger, sqlite/kodi,
                             sqlite/ls_quarantine,
                             sqlite/mac_document_versions,
                             sqlite/mac_knowledgec, sqlite/mac_notes,
                             sqlite/mac_notificationcenter,
                             sqlite/mackeeper_cache, sqlite/safari_history,
                             sqlite/skype, sqlite/tango_android_profile,
                             sqlite/tango_android_tc, sqlite/twitter_android,
                             sqlite/twitter_ios, sqlite/windows_timeline,
                             sqlite/zeitgeist, symantec_scanlog, usnjrnl,
                             winevtx, winfirewall, winjob, winreg,
                             winreg/appcompatcache, winreg/bagmru,
                             winreg/ccleaner, winreg/explorer_mountpoints2,
                             winreg/explorer_programscache,
                             winreg/microsoft_office_mru,
                             winreg/microsoft_outlook_mru,
                             winreg/mrulist_shell_item_list,
                             winreg/mrulist_string,
                             winreg/mrulistex_shell_item_list,
                             winreg/mrulistex_string,
                             winreg/mrulistex_string_and_shell_item,
                             winreg/mrulistex_string_and_shell_item_list,
                             winreg/msie_zone, winreg/mstsc_rdp,
                             winreg/mstsc_rdp_mru, winreg/network_drives,
                             winreg/networks, winreg/userassist,
                             winreg/windows_boot_execute,
                             winreg/windows_boot_verify, winreg/windows_run,
                             winreg/windows_sam_users, winreg/windows_services,
                             winreg/windows_shutdown,
                             winreg/windows_task_cache,
                             winreg/windows_timezone,
                             winreg/windows_typed_urls,
                             winreg/windows_usb_devices,
                             winreg/windows_usbstor_devices,
                             winreg/windows_version, winreg/winlogon,
                             winreg/winrar_mru, winreg/winreg_default
```

(Selection from) Logfile output:

```
 <engine> Preprocessing detected operating systems: Windows NT
 <extraction_tool> Parser filter expression set to: win7
```

**Comment by koromodako** [comment]
I encounter the exact same problem on Ubuntu 18.04 (Python 3.7) with [plaso-20190708](https://github.com/log2timeline/plaso/releases/tag/20190708).

**Comment by Onager** [comment]
This is due to some inverted logic, fix in #2738
